rpc/v1/openapi.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
```

Comment:
You need to run `generate_cli_commands_docs` so the docs are updated

Reply:
`generate_cli_commands_docs` updates docs for the CLI. (this has to be ran in the PR that adds the `non_interactive` flag if I didn't do that already. Noted.

In here I have already ran `make generate_rpc_openapi` before adding the handlers for the new endpoint